ballerina/tests/init-tests.bal

```diff
@@ -289,6 +331,49 @@ public type EmployeeInfo record {|
     Workspace workspace;
 |};
 
+OrderItemExtended orderItemExtended2Retrieved = {
+    orderId: "order-2",
+    itemId: "item-2",
+    CustomerId: 1,
+    paid: false,
+    ammountPaid: 10.5f,
+    ammountPaidDecimal: 10.5,
+    arivalTimeCivil: {"timeAbbrev":"Z","dayOfWeek":5 ,"year":2024,"month":4,"day":12,"hour":12,"minute":20,"second":50.52},
+    arivalTimeUtc: [1684493685, 0.998012000],
+    arivalTimeDate: {year: 2021, month: 4, day: 12},
+    arivalTimeTimeOfDay: {hour: 17, minute: 50, second: 50.52},
+    orderType: ONLINE
+
```

Comment:
```suggestion
```

Reply:
updated in [f7d8c93](https://github.com/ballerina-platform/module-ballerina-persist/pull/218/commits/f7d8c9364ffc8fa17d1d01c211a78e2af887b9bd)